ecs_fargate/tests/test_unit.py

```diff
@@ -0,0 +1,149 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from __future__ import unicode_literals
+import os
+
+import pytest
+import mock
+import simplejson as json
+
+from datadog_checks.ecs_fargate import FargateCheck
+
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture
+def aggregator():
```

Comment:
You should be able to remove this definition of the aggregator as its available as a fixture already here - https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/plugin/plugin.py#L10